File: 2.Traverse/Python/8.MakeAWaveOfArray.py
# Hardcoded input
n = 6
arr = [5, 2, 9, 1, 6, 3]

# Sort the array
arr.sort()

# Swap adjacent elements in pairs
for i in range(0, n - 1, 2):
    arr[i], arr[i + 1] = arr[i + 1], arr[i]

# Print the result
print(*arr)

File: 2.Traverse/Javascript/8.MakeAWaveOfArray.js
// Hardcoded input
let n = 10;
let arr = [9, 8, 4, 10, 3, 6, 5, 7, 1, 2];

// Sort the array
arr.sort((a, b) => a - b);

// Swap adjacent elements in pairs
for (let i = 0; i < n - 1; i += 2) {
    [arr[i], arr[i + 1]] = [arr[i + 1], arr[i]];
}

// Print the result
console.log(...arr);
